content/en/mobile_app_testing/_index.md

```diff
@@ -23,14 +22,19 @@ cascade:
     tags: ['mobile_testing']
 ---
 
-{{< site-region region="us3,us5,gov,eu,ap1" >}}
-<div class="alert alert-warning">Mobile Application Testing is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< site-region region="us,us5,eu" >}}
+<div class="alert alert-warning">Mobile Application Testing is generally available for US1, US5, and EU sites. Support for US3 and AP1 coming soon.</div>
```

Comment:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is Generally Available for US1, US5, and EU sites.</div>
```

Reply:
Suggestion to match existing styling of similar text, and removing promises as a matter of docs policy.